Devscord.DiscordFramework/Middlewares/Contexts/UserRole.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections.Generic;
+using Devscord.DiscordFramework.Commons;
+
+namespace Devscord.DiscordFramework.Middlewares.Contexts
+{
+    public class UserRole
+    {
+        public ulong Id { get; private set; }
+        public string Name { get; private set; }
+        public Permissions Permissions { get; private set; }
+
+        public UserRole(string name)
```

Comment:
czemu ID nie zawsze jest uzupełniane?

Reply:
Ponieważ kiedy tworzę rolę, aby ją stworzyć na serwerze, to wtedy jeszcze nie istnieje jej ID. Dopiero po utworzeniu je dostaje

Reply:
do tego byłby lepszy osobny obiekt -> dedykowany do tworzenia roli